### DISCUSSION TITLE
Slash commands are not working

### DISCUSSION THREAD

**Discussion by master-of-darkness** [DisnakeDev/disnake#494] [Q&A]
### Summary

Slash command not working

### Reproduction Steps

I just created simple slash command and during 2 hours its not appearing

### Minimal Reproducible Code

```python
@bot.slash_command(name="news", description="Only for admins")
async def news(interaction: disnake.ApplicationCommandInteraction):
    print(interaction.author.roles)
    if "Модератор" in interaction.author.roles:
        await interaction.response.send_modal(modal=news.News(bot=bot))
        interaction.response.is_done()
    else:
        interaction.response.is_done()


@bot.slash_command(name="online", description="Получить статус сервера")
async def online(interaction: disnake.ApplicationCommandInteraction):
    await interaction.response.send_message(embed=disnake.Embed(title="Статус сервера", description=f"**Онлайн {mc_status.players.online}/{mc_status.players.max}\n{mc_status.players.sample if mc_status.players.sample != False else 'Никого нет('}```"))
```

### Expected Results

My slash command started working

### Actual Results

It didn't working and nothing happening

### Intents

nothing

### System Information

- Python v3.10.4-final
- disnake v2.4.0-final
    - disnake pkg_resources: v2.4.0
- aiohttp v3.7.4.post0
- system info: Linux 5.16.19-200.fc35.x86_64 #1 SMP PREEMPT Fri Apr 8 15:34:44 UTC 2022

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

I waited 2 hours and my slash command not appearing in commands list and it's not working.Maybe I did smth wrong? I'm sorry for my English cuz I'm from Ukraine. I hope you will help me with my issue

**Comment by ironman9356** [comment]
do you have test_guilds or sync commands in your bot definition ? 
if not try those

**Comment by master-of-darkness** [comment]
> do you have test_guilds or sync commands in your bot definition ? if not try those

While using test_quilds or guild_ids  I getting error SyncWarning: failed to overwrite commands in smbd channel and auto_sync is True by default

**Comment by ironman9356** [comment]
> > do you have test_guilds or sync commands in your bot definition ? if not try those
> 
> While using test_quilds or guild_ids I getting error SyncWarning: failed to overwrite commands in smbd channel and auto_sync is True by default

sorry i meant sync_permissions 
also make sync_commands_debug = True in the bot definition and try again ( also i think the slash commands should be here by now )

**Reply by master-of-darkness** [reply]
> > > do you have test_guilds or sync commands in your bot definition ? if not try those
> > 
> > 
> > While using test_quilds or guild_ids I getting error SyncWarning: failed to overwrite commands in smbd channel and auto_sync is True by default
> 
> sorry i meant sync_permissions also make sync_commands_debug = True in the bot definition and try again ( also i think the slash commands should be here by now )

Here is output of code:

![изображение](https://user-images.githubusercontent.com/46718356/165280060-858537e3-16a4-4ffc-9ec2-0f3a6a3337d0.png)

**Reply by ironman9356** [reply]
basically it works 
whats the issue then ?

**Reply by master-of-darkness** [reply]
It doesn't show up in autocontinue and doesn't work

**Reply by ironman9356** [reply]
ok reinvite your bot with the appilcation commands scope no need to kick it btw
